client/my-sites/site-settings/site-owner-transfer/start-site-owner-transfer.tsx

```diff
@@ -27,18 +33,185 @@ const SiteOwnerTransferActionPanelBody = styled( ActionPanelBody )( {
 	overflow: 'visible !important',
 } );
 
+const Title = styled.h2( {
+	fontWeight: 'bold',
+	marginBottom: '1em',
+} );
+
+const Text = styled.p( {
+	marginBottom: '0.5em !important',
+} );
+
+const List = styled.ul( {
+	marginLeft: '2em',
+} );
+
+const ListItem = styled.li( {
+	marginBottom: '0.25em',
+} );
+
+const DomainsCard = ( {
+	domains,
+	siteSlug,
+	siteOwner,
+}: {
+	domains: ResponseDomain[];
+	siteSlug: string | null;
+	siteOwner?: string;
+} ) => {
+	const translate = useTranslate();
+	return (
+		<>
+			<Title>{ translate( 'Domains' ) }</Title>
+			{ domains.length === 0 ? (
+				<List>
+					<ListItem>
+						{ createInterpolateElement(
+							sprintf(
+								// translators: siteSlug is the current site slug, siteOwner is the user that the site is going to
+								// transer to
+								translate(
+									'The domain name <strong>%(siteSlug)s</strong> will be transferred to <strong>%(siteOwner)s</strong> and will remain working on the site.'
```

Comment:
could we also please set these to ` font-weight: 500` to be consistent with the `const Title = styled.h2`?